plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/procedure/VacuumProcedure.java

```diff
@@ -295,21 +313,32 @@ private void doVacuum(
                     retainedUnknownFiles++;
                     continue;
                 }
-
                 log.debug("[%s] deleting file [%s] with modification time %s", queryId, location, modificationTime);
-                filesToDelete.add(entry.location());
```

Comment:
Can you extract refactoring to preparatory commit, so it's easier to see what is added in feature itself?

Reply:
done